src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -186,6 +186,7 @@ class Repository(
     agencyInternalLocationRepository.findById(locationId).orElse(null)
 
   fun delete(offender: Offender) = offenderRepository.deleteById(offender.id)
+  fun deleteOffenders() = offenderRepository.deleteAll()
```

Comment:
just need to check at some point that any 'static' seed data that gets deleted isnt needed